.travis.yml

```diff
@@ -1,19 +1,24 @@
 sudo: required
+
 services:
 - docker
-language: node_js
-node_js:
-- node
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
 script:
-- npm run lint
-- npm run snyk-test
-- npm run generate-coverage
-- npm run check-coverage
-- npm run upload-coverage-coveralls
+  - docker-compose -f docker-compose-travis.yml up --build -d
```

Comment:
The changes in #40 work (and should be ok to merge in). Extra details in the PR description, it boils down to using `build` and `run` rather than `up`.